source/includes/fact-log-rotation-usr1-signal.rst

```diff
@@ -0,0 +1,8 @@
+If you start ``mongosync`` with the ``--logPath`` option, you can send a
+``USR1`` signal to the ``mongosync`` process to rotate its log file:
+
+.. code-block:: javascript 
+
+   kill -s USR1 $mongosync_pid
+
+Here, ``$mongosync_pid`` is the process ID of the ``mongosync`` process.
```

Comment:
[Nit]

Consider removing "Here". I don't think it's needed. Also leaving a small wording suggestion.

```suggestion
``$mongosync_pid`` is the ``mongosync`` process ID.
```